activestorage/app/models/active_storage/checksum.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ActiveStorage::Checksum
+  require "openssl"
+  require "digest/crc32"
+  require "digest/crc32c"
+  require "digest/crc64"
+
+  SUPPORTED_CHECKSUMS = {
+    MD5: ActiveStorage.md5_checksum_implementation,
+    SHA1: OpenSSL::Digest::SHA1,
+    SHA256: OpenSSL::Digest::SHA256,
+    CRC32: Digest::CRC32,
+    CRC32c: Digest::CRC32c,
+    CRC64: Digest::CRC64
+  }
```

Comment:
This way of doing things impose early loading of `CRC*` even if they're never used.